
export const getState=async(req,res)=>{
	try {
		
	} catch (error) {
		res.status(500).json("internal server error")
	}
}